src/api/v4/dataview/histogram.js

```diff
@@ -0,0 +1,186 @@
+var _ = require('underscore');
+var Base = require('./base');
+var HistogramDataviewModel = require('../../..//dataviews/histogram-dataview-model');
+
+/**
+ * Histogram dataview object
+ *
+ * @param {carto.source.Base} source - The source where the dataview will fetch the data.
+ * @param {string} column - The column name to get the data.
+ * @param {object} options
+ * @param {number} [options.bins=10] - Number of bins to aggregate the data range into.
+ *
+ * @constructor
+ * @extends carto.dataview.Base
+ * @memberof carto.dataview
+ * @api
+ */
+function Histogram (source, column, options) {
+  this._initialize(source, column, options);
+  this._bins = this._options.bins;
+}
+
+Histogram.prototype = Object.create(Base.prototype);
+
+Histogram.prototype.DEFAULTS = {
+  bins: 10
+};
+
+/**
+ * Return the resulting data
+ *
+ * @return {HistogramData}
+ * @api
+ */
+Histogram.prototype.getData = function () {
+  if (this._internalModel) {
+    return this._parseData(this._internalModel.get('data'), this._internalModel.get('nulls'), this._internalModel.get('totalAmount'));
+  }
+  return null;
+};
+
+/**
+ * Return the totals data (not affected by filters)
+ *
+ * @return {HistogramData}
+ * @api
+ */
+
+Histogram.prototype.getTotalsData = function () {
+  if (this._internalModel && this._internalModel.getUnfilteredData()) {
+    var totals = this._internalModel.getUnfilteredDataModel();
+    return this._parseData(this._internalModel.getUnfilteredData(), totals.get('nulls'), totals.get('totalAmount'));
+  }
+  return null;
+};
+
+/**
+ * Set number of bins
+ * 
+ * @param {number} bins
+ * @return {carto.dataview.Histogram} this
+ * @api
+ */
+Histogram.prototype.setBins = function (bins) {
+  this._validateBins(bins);
+  var floorBins = Math.floor(bins);
+  this._changeProperty('bins', floorBins);
+  return this;
+};
+
+/**
+ * Return the current number of bins
+ * 
+ * @return {number} Current number of bins
+ * @api 
+ */
+Histogram.prototype.getBins = function () {
+  return this._bins;
+};
+
+/**
+ * Return the distribution type of the current data according to Galtung’s AJUS System http://druedin.com/2012/12/08/galtungs-ajus-system/
```

Comment:
Isn't there a more "official" website to point to (eg: an entry in the wikipedia)?

Reply:
You're right. I'll link to Wikipedia.